tests/test_project/tests/test_qa.py

```diff
@@ -1,6 +1,5 @@
 import io
```

Comment:
I see 
```
usage: commitcheck [-h] --message MESSAGE [--quiet]
commitcheck: error: the following arguments are required: --message
...usage: checkmigrations [-h] --migration-path MIGRATION_PATH
                       [--migrations-to-ignore MIGRATIONS_TO_IGNORE] [--quiet]
checkmigrations: error: the following arguments are required: --migration-path
```

in the output, can you please supress that as well?

Reply:
Okay, I will

Reply:
I try to supress them as well but seems decorator is not working on argument parser :thinking:

Reply:
I see, could you investigate on this?